{
    "total_points_awarded": 0,
    "tests": [
        {
            "traceback": "AssertionError: There was an error importing your code.\nNo module named 'matplotlib'\n",
            "output": {
                "points_awarded": 0,
                "points_available": 0
            },
            "description": "Test module import",
            "err_msg": "No module named 'matplotlib'"
        }
    ],
    "total_points_available": 0
}